### PR TITLE
Clean up variables -- some renames and a new constant

### PR DESCRIPTION
**Pull request by mklucsarits**
This includes the new FanFeed component added during training with Ludwik.

MainPage -> HomePage was a `git mv` but it shows as a new file and deletion here.

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/elements/fandom/components/HeroBlock.java [hunk 1]
```diff
@@ -16,7 +16,7 @@ public boolean isDisplayed() {
     try {
       return heroMosaic.isDisplayed();
     } catch (ElementNotFoundException e) {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/elements/fandom/components/HeroBlock.java]: MAJOR Either log or rethrow this exception. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1166)

File: src/test/java/com/wikia/webdriver/elements/fandom/FandomPage.java [hunk 2]
```diff
@@ -5,4 +5,6 @@
 public abstract class FandomPage<T> extends BasePageObject {
 
   public abstract T open();
+
+  public static final String SITE_URL = "http://qa.fandom.wikia.com/";
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/elements/fandom/FandomPage.java]: MINOR Move this variable to comply with Java Code Conventions. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1213)

File: src/test/java/com/wikia/webdriver/elements/fandom/components/FanFeed.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+public class FanFeed extends BasePageObject {
+
+  @FindBy(css = "section.fan-feed")
+  private WebElement fanFeed;
+
+  public boolean isDisplayed() {
+    try {
+      return fanFeed.isDisplayed();
+    } catch (ElementNotFoundException e) {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/elements/fandom/components/FanFeed.java]: MAJOR Either log or rethrow this exception. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1166)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues:
* MAJOR 2 major
* MINOR 1 minor

Watch the comments in this conversation to review them.

**Comment by lizlux** [comment]
LGTM. It would be good to get some QA eyes on this too, @qaga @ludwikkazmierczak

**Comment by lizlux** [comment]
Not sure why the build check is failing though

**Pull request merged**